subprojects/performance/src/performanceTest/groovy/org/gradle/performance/regression/corefeature/ParallelDownloadsPerformanceTest.groovy

```diff
@@ -35,6 +36,7 @@ import javax.servlet.ServletResponse
 import javax.servlet.http.HttpServletRequest
 import java.util.concurrent.atomic.AtomicInteger
 
+@Ignore("These tests are dedicated to test external repositories")
```

Comment:
❌ External, yes, but that doesn't mean we can't use our mirrors. IIUC, the test start a local http server which acts as the external repository. We should still use our own mirrors here.

Reply:
Fixed.